server/tests/admin-test-data.ts

```diff
@@ -0,0 +1,28 @@
+import express from 'express';
+
+// Basic Data: Appropriate Admin
+export const AdminRequest1 = {
```

Comment:
for now, could you just move all the data into the admin.test.ts file? currently, we don't have separate data files for other testing files. It's good to have a testing data folder, but we can incorporate that once we have all the tests merged in; otherwise, we would lose consistency across all the tests. up to you tho